### DISCUSSION TITLE
记录：加入绳网第一天

### DISCUSSION THREAD

**Discussion by yinlang520** [share121/inter-knot#3486] [其他]
<img width="72" alt="68747470733a2f2f696d672d7374617469632e6d69686f796f2e636f6d2f636f6d6d756e6974797765622f75706c6f61642f34363034373430383739313761643639613466663937356133313332613064662e706e67" src="https://github.com/user-attachments/assets/8b1c0e23-d3ab-4d28-ac2c-b43195f66931">

**Comment by Arocohol** [comment]
能看到我的回复吗，如果看得到请戳一下我<img src="https://img-static.mihoyo.com/communityweb/upload/ee7bb800b0c90f66fa9e321cb6d8ee17.png" alt="抱大腿" width="75" >

**Reply by getmotos** [reply]
还需要戳吗？

**Reply by Arocohol** [reply]
> 还需要戳吗？
哇谢谢（）
[派蒙_流口水]

**Reply by Arocohol** [reply]
> 还需要戳吗？
<img src="https://img-static.mihoyo.com/communityweb/upload/fe198bb1523528c2b852d28626379a2a.png" alt="心情简单" width="75" >
